### PR TITLE
events section

### PR DESCRIPTION
**Pull request by tr-ann**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -82,8 +76,8 @@
           <img class="app-container__image" src="shared/landing/app-picture.png" alt="Mobile app" />
         </div>
       </section>
-      <section id="howitworks" class="guide">
-        <div class="guide__title">
+      <section id="howitworks" class="guide container container_column">
+        <h1 class="container__title container__title_default">
           <span>How it works</span>
```
Comment (iharzh) [inline index.html]: You don't need span tag here. You can leave just text

File: index.html [hunk 2]
```diff
@@ -129,16 +123,21 @@
...
       </section>
-      <section id="interaction" class="interaction">
-        <div class="interaction__side-title">
+      <section id="interaction" class="interaction container container_row">
+        <div class="container__side-title container__side-title_light">
           <svg height="5rem" width="2rem">
             <rect height="4rem" width="0.07rem" x="0.95rem" y="0.5rem" fill="white" />
             <circle r="0.4rem" cx="1rem" cy="0.5rem" fill="white" />
           </svg>
           <span>Interaction</span>
```
Comment (iharzh) [inline index.html]: You don't need span tag here. You can leave just text

File: index.html [hunk 3]
```diff
@@ -196,13 +195,13 @@
           </div>
         </div>
       </section>
-      <section id="features" class="features">
-        <div class="features__title">
+      <section id="features" class="features container container_column">
+        <h1 class="container__title container__title_default">
           <span>Features</span>
```
Comment (iharzh) [inline index.html]: You don't need span tag here. You can leave just text

File: index.html [hunk 4]
```diff
@@ -129,16 +123,21 @@
...
           <svg height="5rem" width="2rem">
             <rect height="4rem" width="0.07rem" x="0.95rem" y="0.5rem" fill="white" />
             <circle r="0.4rem" cx="1rem" cy="0.5rem" fill="white" />
           </svg>
           <span>Interaction</span>
         </div>
         <div class="interaction__info-container">
-          <div class="info-container__title">Interact with thousand of audience in no time</div>
+          <h2 class="info-container__title">
+            <span class="info-container__title_line">Interact with</span>
```
Comment (iharzh) [inline index.html]: You can put all the text inside `<h2>` and just limit it's width to make it fit multiple lines. No matter if those will not match the design exactly.

File: index.html [hunk 5]
```diff
@@ -316,6 +315,80 @@
           </div>
         </div>
       </section>
+      <section id="events" class="events container container_column">
+        <h1 class="container__title container__title_default">
+          <span>Events</span>
```
Comment (iharzh) [inline index.html]: You don't need span tag here. You can leave just text

File: index.html [hunk 6]
```diff
@@ -316,6 +315,80 @@
...
+              <span class="card__title_bold">event</span>
+            </h1>
+            <div class="container container_column">
+              <img
+                class="card-info__image"
+                src="./shared/events/usb-connection.png"
+                alt="connection"
+              />
+              <div class="card-info__text">
+                <p class="card-info__text_line">Offers customizable on-line registration</p>
```
Comment (iharzh) [inline index.html]: Same thing. You don't have to split the text in lines. Write it as is and limit it's width if necessary

File: index.html [hunk 7]
```diff
@@ -316,6 +315,80 @@
...
+            </svg>
+            <svg width="0.5rem" height="0.2rem">
+              <rect width="0.5rem" height="0.2rem" rx="0.1rem" fill="var(--color--events-title)" />
+            </svg>
+          </div>
+        </h1>
+        <div class="events__cards-container container container_row">
+          <div class="events__card">
+            <h1 class="card__title card__title_external-event-bg">
+              <span>external</span>
```
Comment (iharzh) [inline index.html]: You don't need span tag here. You can leave just text

File: index.html [hunk 8]
```diff
@@ -316,6 +315,80 @@
...
+                <p class="card-info__text_line">Offers customizable on-line registration</p>
+                <p class="card-info__text_line">service for event organizers. Includes</p>
+                <p class="card-info__text_line">pricing, features, and types of accounts</p>
+              </div>
+              <a class="card-info__button card-info__button_success" href="#">Discover</a>
+            </div>
+          </div>
+          <div class="events__card">
+            <h1 class="card__title card__title_interactive-event-bg">
+              <span>interactive</span>
```
Comment (iharzh) [inline index.html]: You don't need span tag here. You can leave just text

File: index.html [hunk 9]
```diff
@@ -316,6 +315,80 @@
...
+                <p class="card-info__text_line">Offers customizable on-line registration</p>
+                <p class="card-info__text_line">service for event organizers. Includes</p>
+                <p class="card-info__text_line">pricing, features, and types of accounts</p>
+              </div>
+              <a class="card-info__button card-info__button_danger" href="#">Discover</a>
+            </div>
+          </div>
+          <div class="events__card">
+            <h1 class="card__title card__title_internal-event-bg">
+              <span>internal</span>
```
Comment (iharzh) [inline index.html]: You don't need span tag here. You can leave just text

File: index.html [hunk 10]
```diff
@@ -316,6 +315,80 @@
...
+              <span class="card__title_bold">event</span>
+            </h1>
+            <div class="container container_column">
+              <img
+                class="card-info__image"
+                src="./shared/events/interactivity.png"
+                alt="connection"
+              />
+              <div class="card-info__text">
+                <p class="card-info__text_line">Offers customizable on-line registration</p>
```
Comment (iharzh) [inline index.html]: You don't have to split the text in lines. Write it as is and limit it's width if necessary

File: index.html [hunk 11]
```diff
@@ -316,6 +315,80 @@
...
+          </div>
+          <div class="events__card">
+            <h1 class="card__title card__title_internal-event-bg">
+              <span>internal</span>
+              <span class="card__title_bold">event</span>
+            </h1>
+            <div class="container container_column">
+              <img class="card-info__image" src="./shared/events/hard-disk.png" alt="connection" />
+              <div class="card-info__text">
+                <p class="card-info__text_line">Offers customizable on-line registration</p>
```
Comment (iharzh) [inline index.html]: You don't have to split the text in lines. Write it as is and limit it's width if necessary

File: styles/common.css [hunk 12]
```diff
@@ -1,5 +1,15 @@
 @import 'colors.css';
 @import 'underlines/underlines.css';
+@import 'container/container.css';
+@import 'container/title.css';
+
+@import 'header/header.css';
```
Comment (iharzh) [inline styles/common.css]: Styles for sections are not common styles. If you want to have a main css file for imports - then you can create `main.css` and import everything there.

File: styles/common.css [hunk 13]
```diff
@@ -17,7 +27,22 @@ a {
...
+h2 {
+  font-size: 2.5rem;
+}
+
 @font-face {
   font-family: 'Segoe';
   src: url('../shared/fonts/segoe_ui.ttf');
 }
+
+@font-face {
```
Comment (iharzh) [inline styles/common.css]: Are you sure that we are using two different fonts here?

File: styles/container/title.css [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+.container__title_default {
+  color: var(--color--secondary-text);
+}
+
+.container__side-title {
+  display: flex;
+  flex-flow: row nowrap;
+
```
Comment (iharzh) [inline styles/container/title.css]: Please remove empty lines between properties

**Review by iharzh — CHANGES_REQUESTED** [review]

File: styles/common.css (hunk 13 above)
Comment (tr-ann) [reply styles/common.css:45]: i'm sure)

**Review by iharzh — APPROVED** [review]

**Pull request merged**